### PR TITLE
biblioteq: Add version 2021.02.12

### PR DESCRIPTION
**Pull request by janlindblom**
Add an installer for BiblioteQ (a library management app).

### CODE REVIEW

File: bucket/biblioteq.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    ],
+    "homepage": "https://textbrowser.github.io/biblioteq/",
+    "version": "2020.09.09",
+    "hash": "6e488066631654d9ce87793098dc4659128fc4fd03c747a160467c320a48740b",
+    "autoupdate": {
+        "url": "https://github.com/textbrowser/biblioteq/releases/download/$version/BiblioteQ.zip"
+    },
+    "checkver": {
+        "github": "https://github.com/textbrowser/biblioteq"
+    }
```
Comment (Ash258) [inline bucket/biblioteq.json:20]:
```suggestion
    "version": "2020.09.09",
    "description": "Professional archiving, cataloging, and library management suite providing connectivity to PostgreSQL and SQLite.",
    "homepage": "https://textbrowser.github.io/biblioteq/",
    "license": "BSD",
    "url": "https://github.com/textbrowser/biblioteq/releases/download/2020.09.09/BiblioteQ.zip",
    "hash": "6e488066631654d9ce87793098dc4659128fc4fd03c747a160467c320a48740b",
    "bin": "BiblioteQ\\BiblioteQ.exe",
    "shortcuts": [
        [
            "BiblioteQ\\BiblioteQ.exe",
            "BiblioteQ"
        ]
    ],
    "checkver": {
        "github": "https://github.com/textbrowser/biblioteq"
    },
    "autoupdate": {
        "url": "https://github.com/textbrowser/biblioteq/releases/download/$version/BiblioteQ.zip"
    }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/biblioteq.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "license": "BSD",
+    "url": "https://github.com/textbrowser/biblioteq/releases/download/2020.09.09/BiblioteQ.zip",
+    "hash": "6e488066631654d9ce87793098dc4659128fc4fd03c747a160467c320a48740b",
+    "bin": "BiblioteQ\\BiblioteQ.exe",
+    "shortcuts": [
+        [
+            "BiblioteQ\\BiblioteQ.exe",
+            "BiblioteQ"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/biblioteq.json:14]:
```suggestion
    ],
    "persist": ".biblioteq",
```

File: bucket/biblioteq.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "2020.09.09",
+    "description": "Professional archiving, cataloging, and library management suite providing connectivity to PostgreSQL and SQLite.",
+    "homepage": "https://textbrowser.github.io/biblioteq/",
+    "license": "BSD",
+    "url": "https://github.com/textbrowser/biblioteq/releases/download/2020.09.09/BiblioteQ.zip",
+    "hash": "6e488066631654d9ce87793098dc4659128fc4fd03c747a160467c320a48740b",
+    "bin": "BiblioteQ\\BiblioteQ.exe",
+    "shortcuts": [
+        [
+            "BiblioteQ\\BiblioteQ.exe",
```
Comment (Ash258) [inline bucket/biblioteq.json:11]:
```suggestion
    "extract_dir": "BiblioteQ",
    "pre_install": "Remove-Item \"$dir\\vc_red*\"",
    "bin": "BiblioteQ.exe",
    "shortcuts": [
        [
            "BiblioteQ.exe",
```

File: bucket/biblioteq.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2020.09.09",
+    "description": "Professional archiving, cataloging, and library management suite providing connectivity to PostgreSQL and SQLite.",
+    "homepage": "https://textbrowser.github.io/biblioteq/",
+    "license": "BSD",
+    "url": "https://github.com/textbrowser/biblioteq/releases/download/2020.09.09/BiblioteQ.zip",
+    "hash": "6e488066631654d9ce87793098dc4659128fc4fd03c747a160467c320a48740b",
```
Comment (Ash258) [inline bucket/biblioteq.json:7]:
```suggestion
    "suggest": {
        "vcredist": "vcredist2015"
    },
    "url": "https://github.com/textbrowser/biblioteq/releases/download/2021.02.12/BiblioteQ.zip",
    "hash": "96c8ef586db260bcda3f3ec7c7883c04e8cb4179f69bfdbcb72f6584b9a38d8c"
```

File: bucket/biblioteq.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2020.09.09",
```
Comment (Ash258) [inline bucket/biblioteq.json:2]:
```suggestion
    "version": "2021.02.12",
```

File: bucket/biblioteq.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    "description": "Professional archiving, cataloging, and library management suite providing connectivity to PostgreSQL and SQLite.",
+    "homepage": "https://textbrowser.github.io/biblioteq/",
+    "license": "BSD",
```
Comment (Ash258) [inline bucket/biblioteq.json:5]: Invalid SPDX identifier

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/biblioteq.json (hunk 6 above)
Comment (janlindblom) [reply bucket/biblioteq.json:5]: Ok will fix, it's the bsd 3-clause license.

Comment (janlindblom) [reply bucket/biblioteq.json]: SPDX identifier BSD-3-Clause now added.

**Comment by janlindblom** [comment]
Should be good now, thanks for the help @Ash258 !

**Pull request merged**